.storybook/main.js

```diff
@@ -142,7 +142,7 @@ const arWebpack = (config) => {
 	});
 
 	config.resolve.modules = [
-		...(config?.resolve?.modules || []),
+		...(config && config.resolve && config.resolve.modules || []),
```

Comment:
Is this is needed? I'm under the impression that the version of node in the `.nvmrc` supports this syntax

Reply:
Hey Olly, good question. This was breaking when I ran `yarn chromatic` and I think @nitro-marky was having an issue with this the other day also.